tests/test_create_samples.py

```diff
@@ -0,0 +1,293 @@
+from unittest.mock import patch, Mock
+
+from pyclarity_lims.entities import Sample
+
+from tests.test_common import TestEPP, FakeEntitiesMaker
+
+from scripts.create_samples import CreateSamples, Container
+from tests.test_common import TestEPP, FakeEntitiesMaker
+
+
+
+class TestCreateSamples(TestEPP):
```

Comment:
Not sure what happened during the merge but it looks like everything got duplicated

Reply:
updated